modules/apps/headless-apio/folder/folder-apio-client-test/src/testIntegration/java/com/liferay/folder/apio/client/test/FolderApioTest.java

```diff
@@ -101,6 +101,157 @@ public void testCreateFolder() {
 		);
 	}
 
+	@Test
+	public void testCreateSubfolder() {
+		String path = ApioClientBuilder.given(
+		).basicAuth(
+			"test@liferay.com", "test"
+		).header(
+			"Accept", "application/hal+json"
+		).when(
+		).get(
+			_rootEndpointURL.toExternalForm()
+		).follow(
+			"_links.content-space.href"
+		).follow(
+			"_embedded.ContentSpace.find {it.name == '" +
+				FolderTestActivator.CONTENT_SPACE_NAME +
+					"'}._links.documentsRepository.href"
+		).then(
+		).extract(
+		).path(
+			"_links.folders.href"
+		);
+
+		ApioClientBuilder.given(
+		).basicAuth(
+			"test@liferay.com", "test"
+		).header(
+			"Accept", "application/hal+json"
+		).header(
+			"Content-Type", "application/json"
+		).body(
+			"{\"description\":\"Example description 3\",\"name\":\"Example " +
```

Comment:
Maybe Brian would tell you: Where there is a 3, a 2 and 1  should be exist also.